server/sonar-web/src/main/js/apps/coding-rules/rule/profile-activation-view.js

```diff
@@ -110,21 +110,21 @@ export default ModalForm.extend({
         // do not show global error
         400: null
       }
-    }).done(function () {
+    }).done(() => {
       that.destroy();
       that.trigger('profileActivated', severity, params, profileKey);
-    }).fail(function (jqXHR) {
+    }).fail(jqXHR => {
       that.enableForm();
       that.showErrors(jqXHR.responseJSON.errors, jqXHR.responseJSON.warnings);
     });
   },
 
   getAvailableQualityProfiles (lang) {
     const activeQualityProfiles = this.collection || new Backbone.Collection();
-    const inactiveProfiles = _.reject(this.options.app.qualityProfiles, function (profile) {
+    const inactiveProfiles = _.reject(this.options.app.qualityProfiles, profile => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)